src/bin/pubsub_server.rs

```diff
@@ -0,0 +1,169 @@
+extern crate actix_web;
+#[macro_use]
+extern crate log;
+extern crate actix;
+extern crate env_logger;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate clap;
+extern crate futures;
+extern crate pubsub;
+extern crate serde;
+
+use actix_web::State as ActixState;
+use actix_web::{
+    http::Method, middleware::Logger, server, App, AsyncResponder, Error, HttpMessage, HttpRequest,
+    HttpResponse, Path,
+};
+use clap::{App as ClApp, Arg};
+use futures::future::Future;
+use pubsub::{ConnectionInfo, PublisherDesc};
+use std::collections::HashMap;
+use std::net::ToSocketAddrs;
+use std::option::Option;
+use std::sync::{Arc, RwLock};
+use std::time;
+
+#[derive(Serialize, Debug)]
```

Comment:
These responses will probably be moved to a more central location in the library before long.